olive/engine/config.py

```diff
@@ -32,3 +33,4 @@ class EngineConfig(ConfigBase):
     clean_cache: bool = False
     clean_evaluation_cache: bool = False
     plot_pareto_frontier: bool = False
+    auto_optimizer_config: Optional[AutoOptimizerConfig] = None
```

Comment:
should we move the config out of engine? I suppose it should be above the engine level?

Reply:
Updated.